torch/_dynamo/variables/misc.py

```diff
@@ -900,9 +908,9 @@ def wraps(fn):
                 path = inspect.getfile(self.value)
             except TypeError:
                 path = f"Builtin {self.value.__name__}"
-            unimplemented(
-                f"'call_function {self.value.__qualname__} in skip_files {path}, {self.reason}'"
-            )
+            msg = f"'skip function {self.value.__qualname__} in file {path}'"
+            msg += f"', {self.reason}'" if self.reason else ""
+            unimplemented(msg)
```

Comment:
The item in ```manual_torch_name_rule_map``` doesn't have ```reason``` filled, so leaving it empty. Or I can make ```reason``` default by ```according rule in manual_torch_name_rule_map```.